sale_service_project/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+====================
+Sale Service Project
+====================
+
+This module extends the functionality of sale workflow and allow you to create
+projects and tasks with materials and works from sale lines.
+You can invoice from sale order or analytic lines.
+
+Usage
+=====
+
+To use this module, you need to:
+
+1. Create a service product and check 'Create Task Automatically'.
+2. Now you can add task works and materials for this product.
+3. You can compute the total price based in all materials and time.
```

Comment:
Sorry, I misunderstood with hr_timesheet_invoice. Anyway, you should indicate that on each product, you can add materials and operations.